projects/ion/src/lib/tab/tab.component.html

```diff
@@ -0,0 +1,14 @@
+<button
+  (click)="select()"
+  [disabled]="disabled"
+  [ngClass]="{
+    tab: !selected,
+    'tab-selected': selected,
+    'tab-sm': tabSize === 'sm',
+    'tab-md': tabSize === 'md',
+    'tab-lg': tabSize === 'lg'
+  }"
+  [class]="'border-' + direction"
+>
+  {{ label }}
```

Comment:
in this situation the label will not give any error if you do not pass the label through the attribute.
there are two ways to adjust this

1° in the html file
`{{ label || 'Configure uma label' }}`

2° in the ts file
`@Input() label = "Configure uma label"`